daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/v2/ledgerinteraction/SubmitError.scala

```diff
@@ -361,6 +361,15 @@ object SubmitError {
       )
   }
 
+  final case class ValueNesting(limit: Int) extends SubmitError {
+    override def toDamlSubmitError(env: Env): SValue =
+      SubmitErrorConverters(env).damlScriptError(
+        "ValueNesting",
+        18,
```

Comment:
Not 100% that incrementing and bumping rank numbers is the correct way to do this?

Reply:
@samuel-williams-da ?

Reply:
You need to add the new variant to `daml-script/daml3/Daml/Script/Questions/Submit/Error.daml` in the same position.